syntax = "proto3";

message VideoData {
  string video_url = 1;
  string transcript = 2;
  string channel = 3;
  string publication_date = 4;
  string category = 5;
}
